modules/azure/function_app_windows/variables.tf

```
@@ -111,3 +111,9 @@ variable "use_32_bit_worker" {
   description = "Should the Windows Function App use a 32-bit worker process."
   default     = true
 }
+
+variable "app_scale_limit" {
+  type        = number
+  description = "Number of workers this function app can scale out to. Only applicable to apps on the Consumption and Premium plan."
+  default     = 1
```

Have you tested the difference between no `app_scale_limit` vs `app_scale_limit` with value `1`?

Should the default not be
```suggestion
  default     = null
```

Please test this first

---

Just finished some tests  and did necessary code changes so its not mandatory and can be set to null or 0 which is having same unlimited effect